tests/php/Models/Extensions/ExternalLinkExtension.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace SilverStripe\LinkField\Tests\Models\Extensions;
```

Comment:
```suggestion
namespace SilverStripe\LinkField\Tests\Extensions;
```
And move the `Extensions/` directory out of the `Models/` directory.

Reply:
Updated